code/_globalvars/lists/mobs.dm

```diff
@@ -1,8 +1,7 @@
 GLOBAL_LIST_EMPTY(clients)							//all clients
-GLOBAL_LIST_EMPTY(admins)							//all clients whom are admins
+GLOBAL_LIST_EMPTY_TYPED(admins, /datum/admins)							//all clients whom are admins
```

Comment:
There's probably a better solution than making these typed but it gets dreamchecker to shut up.